scripts/format.js

```diff
@@ -1,6 +1,32 @@
+import ParseRss from 'rss-parser'
 import xmlFormat from 'xml-formatter'
-import { getFeedContent, overwriteFeedContent } from '../utils/index.js'
+import {
+  composeFeedItem,
+  getConfig,
+  getFeedContent,
+  overwriteFeedContent,
+} from '../utils/index.js'
+
+const { breakDelimiter } = getConfig()
+
+const parser = new ParseRss()
 
 const xml = getFeedContent()
-const formattedXml = xmlFormat(xml, { indentation: '  ', collapseContent: true })
-overwriteFeedContent(formattedXml)
+
+parser.parseString(xml).then((parsedXml) => {
+  const sortedItems = parsedXml.items
+    .sort((a, b) => (new Date(a.isoDate) < new Date(b.isoDate) ? 1 : -1))
```

Comment:
You can do new Date(a.isoDate) - new Date(b.isoDate)

Reply:
```suggestion
    .sort((a, b) => new Date(a.isoDate) - new Date(b.isoDate))
```